src/mean.jl

```diff
@@ -20,8 +20,7 @@ end
 
 """
     mean(x, design)
-
-Compute the mean and SEM of the survey variable `x`.
+Estimate the population mean of a variable of a simple random sample, and the corresponding standard error.
```

Comment:
```suggestion
Compute the mean and SEM of the survey variable `x`.
```

Reply:
While it's less concise, I think it's okay to give a little more information to make the sentence complete.